### PR TITLE
Update walk and balance images in Motor Control module

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Updated the images, tweeked the storyboard views, and added an option to always show the full instructions as a study configuration.

This should unblock Dan.

### CODE REVIEW

File: Research/Research/RSDStudyConfiguration.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+/// The study configuration is intended as a shared singleton that contains any information that should be
+/// applied to the entire study which may effect the presentation of a given module.
+open class RSDStudyConfiguration {
+    
+    /// Singleton for the study configuration for a given app.
+    public static var shared: RSDStudyConfiguration = RSDStudyConfiguration()
+    
+    /// Should each run of the app display full instructions or can the instructions be abbreviated for
+    /// subsequent runs.
```
Comment (Erin-Mounts) [inline Research/Research/RSDStudyConfiguration.swift]: should be a question mark

File: Research/Research/RSDStudyConfiguration.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import Foundation
+
+/// The study configuration is intended as a shared singleton that contains any information that should be
+/// applied to the entire study which may effect the presentation of a given module.
+open class RSDStudyConfiguration {
+    
+    /// Singleton for the study configuration for a given app.
+    public static var shared: RSDStudyConfiguration = RSDStudyConfiguration()
+    
+    /// Should each run of the app display full instructions or can the instructions be abbreviated for
```
Comment (Erin-Mounts) [inline Research/Research/RSDStudyConfiguration.swift]: maybe a comma after 'full instructions'

**Review by Erin-Mounts — APPROVED** [review]
Couple of tiny nits, otherwise looks good.

**Review by DanWebster — APPROVED** [review]
Overall looks great. Animations for MCT (walk and balance and 30-second walk) were loading somewhat slowly and introducing lag between screens. Didn't have the same issue with the CRF modules. Can't rule out the simulator as a cause, and can test this on my phone tomorrow.  Will follow up on this, but I would not block the merge based on that. Link to screencast below if helpful: https://www.dropbox.com/s/lmacg2i2sc8czfh/pr79.mov?dl=0

**Comment by DanWebster** [comment]
Sidenote that I had no problems building the catalog from scratch and running despite Travis fail

**Pull request merged**